### PR TITLE
In progress course run tabs.

### PR DESCRIPTION
**Pull request by awais786**
Implement Tab for List of In-progress Courses.
ECOM-6038

### CODE REVIEW

**Comment by awais786** [comment]
@tasawernawaz Please review this PR.

**Review by tasawernawaz — APPROVED** [review]

**Comment by awais786** [comment]
@edx/ecommerce Please review this PR.

**Comment by awais786** [comment]
screen shot 2016-11-18 at 10 15 23 pm

**Comment by awais786** [comment]
@bderusha please review this PR.

File: course_discovery/templates/publisher/dashboard.html [hunk 1]
```diff
@@ -7,19 +7,18 @@
 {% endblock title %}
 
 {% block content %}
-    {% with studio_count=studio_request_courses|length published_count=published_course_runs|length preview_count=preview_course_runs|length %}
+    {% with studio_count=studio_request_courses|length published_count=published_course_runs|length preview_count=preview_course_runs|length in_progress_count=in_progress_course_runs|length %}
```
Comment (bderusha) [inline course_discovery/templates/publisher/dashboard.html:10]: Will django template let you break this into a multiline?  If so please do, for readability

File: course_discovery/templates/publisher/dashboard.html [hunk 2]
```diff
@@ -7,19 +7,18 @@
...
     <div class="publisher-container">
         <h2 class="hd-2 emphasized">{% trans "Course runs" %}</h2>
         <ul role="tablist" class="tabs">
-            <li role="tab" id="tab-progress" class="tab" aria-selected="true" aria-expanded="false" aria-controls="progress" tabindex="0"><span>0</span>{% trans "IN PROGRESS" %}</li>
+            <li role="tab" id="tab-progress" class="tab" aria-selected="true" aria-expanded="false" aria-controls="progress" tabindex="0"><span>{{ in_progress_count }}</span>{% trans "IN PROGRESS" %}</li>
```
Comment (bderusha) [inline course_discovery/templates/publisher/dashboard.html]: Readability:  Can we multiline these `li`s as well?

```
<li id="tab"... etc>
    <span>IN PROGRESS...etc</span>
</li>

**Review by bderusha — APPROVED** [review]
A few tiny requests.  Otherwise 👍

File: course_discovery/templates/publisher/dashboard.html (hunk 1 above)
Comment (awais786) [reply course_discovery/templates/publisher/dashboard.html:10]: nop

**Pull request merged**